internal/telemetry/event.go

```diff
@@ -0,0 +1,66 @@
+package telemetry
+
+import (
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+// Event is a telemetry event
+type Event struct {
```

Comment:
does `Event` need to be exported at this point?

Reply:
IDK if it needs to be, but it is the return value of an exported function so I think generally it is a good practice to export it.  Do you want me to un-export it?

Reply:
if `NewEvent` goes away with the refactor to `telemetry.Tracker` below, is it still the return value of an exported function?

Reply:
oh gotcha, I missed that at first